components/director/internal/domain/application/resolver.go

```diff
@@ -741,10 +744,14 @@ func (r *Resolver) EventingConfiguration(ctx context.Context, obj *graphql.Appli
 	if obj == nil {
 		return nil, errors.New("Application cannot be empty")
 	}
-
-	appID, err := uuid.Parse(obj.ID)
+	tenantID, err := tenant.LoadFromContext(ctx)
 	if err != nil {
-		return nil, errors.Wrap(err, "while parsing application ID as UUID")
+		return nil, errors.New("Error while loading tenant from context")
```

Comment:
```suggestion
		return nil, errors.New("error while loading tenant from context")
```

https://github.com/golang/go/wiki/CodeReviewComments#error-strings